app/controlplane/pkg/biz/orgmetrics.go

```diff
@@ -48,11 +48,17 @@ type ByStatusCount struct {
 	Count  int32
 }
 
+// TimeWindow represents in time.Time format not in time.Duration
+type TimeWindow struct {
+	StartDate time.Time
+	EndDate   time.Time
+}
+
 func NewOrgMetricsUseCase(r OrgMetricsRepo, l log.Logger) (*OrgMetricsUseCase, error) {
 	return &OrgMetricsUseCase{logger: log.NewHelper(l), repo: r}, nil
 }
 
-func (uc *OrgMetricsUseCase) RunsTotal(ctx context.Context, orgID string, timeWindow time.Duration) (int32, error) {
+func (uc *OrgMetricsUseCase) RunsTotal(ctx context.Context, orgID string, timeWindow TimeWindow) (int32, error) {
 	orgUUID, err := uuid.Parse(orgID)
 	if err != nil {
 		return 0, err
```

Comment:
we should probably add a timewindow validation, that `From` and `To` is set.

Reply:
Adding them.